File: snippets/fsharp/System/String/Concat/Concat6.fs
module Concat6

// <Snippet6>
open System

let s1 = "We went to a bookstore, "
let s2 = "a movie, "
let s3 = "and a restaurant."

String.Concat(s1, s2, s3)
|> printfn "%s"
// The example displays the following output:
//      We went to a bookstore, a movie, and a restaurant. 
// </Snippet6>

File: snippets/csharp/System/String/Concat/Concat6.cs
// <Snippet6>
using System;

public class ConcatThreeStringsExample
{
    public static void Run()
    {
        string s1 = "We went to a bookstore, ";
        string s2 = "a movie, ";
        string s3 = "and a restaurant.";

        string s = string.Concat(s1, s2, s3);
        Console.WriteLine(s);
    }
}
// The example displays the following output:
//      We went to a bookstore, a movie, and a restaurant.
// </Snippet6>
